src/main/java/io/reactivex/observers/TestObserver.java

```diff
@@ -678,6 +751,65 @@ public final boolean awaitTerminalEvent(long duration, TimeUnit unit) {
     }
 
     /**
+     * Sets the initial fusion mode if the upstream supports fusion.
+     * @param mode the mode to establish, see the {@link QueueDisposable} constants
+     * @return this
+     */
+    public final TestObserver<T> setInitialFusionMode(int mode) {
```

Comment:
All these methods leak an implementation detail of the library into its public API. Granted it's not the Observable API or anything, but it still feels like they don't belong here. Can it be accomplished in an RxJava-specific subclass that lives in the test source set?